fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -121,6 +122,15 @@ void init() throws IOException {
 
         LOGGER.trace("Velocity engine initializing...");
         final Properties properties = new Properties();
+        final String fcrepoHome = getProperty("fcrepo.home");
+        final String velocityLog = getProperty("fcrepo.velocity.runtime.log");
```

Comment:
the issue here is that `fcrepo.home` isn't always set (in which case it defaults to the current directory), and if `fcrepo.velocity.runtime.log` isn't set, it can't be just a directory -- it needs to be a full path to a file, so it would need to be more like this:

    final String velocityLog = getProperty("fcrepo.velocity.runtime.log", getProperty("fcrepo.home", "") + "velocity.log")

but there, if the `fcrepo.home` doesn't end with a file.separator, then you get an incorrect path for the log. And if you put in the separator but you have an empty `fedora.home` value, you may end up with `velocity.log` in the root of the filesystem.

Which is to say, that I could shorten this, but this is pretty readable as it is.